docs/MIGRATION.md

```
@@ -21,247 +21,240 @@ These files may contain **plaintext secrets**:
+### Step 1: Generate Migration Plan
 
 ```bash
-clawvault openclaw migrate --verbose
+clawvault openclaw migrate --plan --verbose
 ```
 
-**[Screenshot Placeholder: Dry-run output showing discovered secrets]**
+This scans all auth stores and generates `clawvault-migration-plan.json`.
 
 Example output:
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)